sdk/src/utils.ts

```diff
@@ -174,15 +174,7 @@ export const deployInvariant = async (
   initFee: Percentage,
   network: Network
 ): Promise<Invariant> => {
-  return Invariant.getContract(
-    api,
-    account,
-    null,
-    DEFAULT_REF_TIME,
-    DEFAULT_PROOF_SIZE,
-    initFee,
-    network
-  )
+  return Invariant.getContract(api, network, account, initFee)
```

Comment:
I'm wondering about the purpose of these functions. All they do is copy the parameters for the `getContract` call. There are two solutions. Choose the better one:
1. Since all `deploy[contract]` functions are used only for tests, and for all tests, we use the `localnet`, you can remove the `Network` parameter and pass `Network.Local` instead.
2. Delete `deploy[contract]` and use `getContract`.